setup.py

```diff
@@ -0,0 +1,176 @@
+"""The setup module for pysimulationio
+"""
+
+from setuptools import setup, find_packages
+from setuptools.extension import Extension
+from distutils.sysconfig import get_python_lib, get_python_inc
+from distutils.command.build import build
+from setuptools.command.install import install
+from distutils.command.clean import clean
+from os import path, environ
+import os, glob
+from shutil import rmtree
+
+# access some simple stuff
+here = path.abspath(path.dirname(__file__))
+py_include_dirs=list(set([get_python_inc(i) for i in range(2)]
+                         +[get_python_lib(i,j)\
+                           for i,j in zip(range(2),range(2))]))
+
+# Read in the package information generated by the makefile.
+with open('VERSION','r') as f:
+    version = float(f.read().strip())
+
+with open('README.md','r') as f:
+    long_description = f.read()
+
+try:
+    with open('includes.txt','r') as f:
+        includes = [x.strip() for x in f.readlines()]
+    includes += py_include_dirs
+    includes = [path.realpath(x) for x in includes]
+
+    with open('flags.txt','r') as f:
+        flags = f.read().strip().split(' ')
+
+    with open('links.txt','r') as f:
+        links = [x.strip() for x in f.readlines()]
+    links = [path.realpath(x) for x in links]
+    link_args = ["-Wl,-rpath,{}".format(p) for p in links]
+
+    with open('compiler.txt','r') as f:
+        cc = f.read().strip()
+    environ["CC"] = cc
```

Comment:
Not an oversight. Python's build system ALWAYS reads the "CC" environment variable. Because setup.py is terrible. I can change the variable names to be more clear, however.